psyneulink/library/compositions/compiledoptimizer.py

```diff
@@ -116,143 +117,143 @@ def step(self, ctx):
-                        weights_llvmlite, [zero, weight_row, weight_column])
+        #     m_t_ptr = builder.gep(
+        #         m_t, [zero, node_idx_ir, afferent_node_index_ir])
+        #     v_t_ptr = builder.gep(
+        #         v_t, [zero, node_idx_ir, afferent_node_index_ir])
+        #     delta_w_ptr = builder.gep(
+        #         delta_w, [zero, node_idx_ir, afferent_node_index_ir])
+        #     # this is messy - #TODO - cleanup this
+        #     weights_llvmlite, weights_dim_x, weights_dim_y = self._pytorch_model._gen_get_node_weight_ptr(
+        #         ctx, builder, params, node, afferent_node)
+        #     pnlvm.helpers.printf(
+        #         builder, f"OPTIM UPDATE WEIGHTS {afferent_node.name} {node.name}\n",override_debug=False)
+        #     weight_row = None
+        #     with pnlvm.helpers.for_loop_zero_inc(builder, ctx.int32_ty(weights_dim_x), "optimizer_w_upd_outer") as (b1, weight_row):
+        #         weight_column = None
+        #         with pnlvm.helpers.for_loop_zero_inc(b1, ctx.int32_ty(weights_dim_y), "optimizer_w_upd_inner") as (b2, weight_column):
+        #             # sqrt(v_t) + eps
+        #             v_t_value = b2.load(b2.gep(
+        #                 v_t_ptr, [zero, weight_row, weight_column]))
+        #             value = b2.call(sqrt, [v_t_value])
+        #             value = b2.fadd(value, eps)
+
+        #             # alpha_t * m_t
+        #             m_t_value = b2.load(b2.gep(
+        #                 m_t_ptr, [zero, weight_row, weight_column]))
+        #             m_t_value = b2.fmul(alpha_t, m_t_value)
+
+        #             # value = alpha_t * m_t / (sqrt(v_t) + eps)
+        #             value = b2.fdiv(m_t_value, value)
+
+        #             old_weight_ptr = b2.gep(
+        #                 weights_llvmlite, [zero, weight_row, weight_column])
                     
-                    # new_weight = old_weight - value
-                    value = b2.fsub(b2.load(old_weight_ptr), value)
-                    b2.store(value, old_weight_ptr)
+        #             # new_weight = old_weight - value
+        #             value = b2.fsub(b2.load(old_weight_ptr), value)
+        #             b2.store(value, old_weight_ptr)
 
-                    delta_w_val = b2.load(b2.gep(delta_w_ptr,[zero, weight_row, weight_column]))
-                    pnlvm.helpers.printf(b2,"%f ",delta_w_val,override_debug=False)
-                pnlvm.helpers.printf(b1,"\n",override_debug=False)
+        #             delta_w_val = b2.load(b2.gep(delta_w_ptr,[zero, weight_row, weight_column]))
+        #             pnlvm.helpers.printf(b2,"%f ",delta_w_val,override_debug=False)
+        #         pnlvm.helpers.printf(b1,"\n",override_debug=False)
                 
-        pnlvm.helpers.printf(builder, f"\t\t\tOPTIM DONE UPDATE\n",override_debug=False)
+        # pnlvm.helpers.printf(builder, f"\t\t\tOPTIM DONE UPDATE\n",override_debug=False)
```

Comment:
This is a lot of dead code. Do we need to keep it around?

Reply:
this was commented out in error - it should not be so (will revert)